src/pages/Main/Components/ProjectCards.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useEffect, useRef } from 'react';
+import Button from '../../../components/Button/Button';
+import ProjectCard from '../../../components/ProjectCard/ProjectCard';
+import { useFilterContext } from '../Main';
+import styled from 'styled-components/macro';
+
+const SORT = [
+  {
+    url: 'Latest',
+    kor: '최신순',
+  },
+  {
+    url: 'People',
+    kor: '최다 후원순',
+  },
+  {
+    url: 'Amount',
+    kor: '최다 금액순',
+  },
+  {
+    url: 'Old',
+    kor: '마감 임박순',
+  },
+];
+
+export default function ProjectCards({ getFilteredData }) {
+  const { filterOption } = useFilterContext();
+  const [mainData, setMainData] = useState([]);
+  const [isOpened, setIsOpened] = useState(false);
+  const [clickedSortOption, setClickedSortOption] = useState(SORT[0]);
+  const nextCount = useRef(6);
```

Comment:
```suggestion
  const nextCountRef = useRef(6);
```
저는 ref의 경우 ref임을 알려줄 수 있는 네이밍을 쓰는게 좋다고 생각해요!

Reply:
아항~넵!